src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
There is a logical error in the nth-child selectors. The selector '.stars--0 .stars__star:nth-child(-n + 0)' will not select any stars because '-n + 0' results in no positive integers. You should adjust the nth-child selectors to correctly target the intended number of stars for each '.stars--X' class. For example, '.stars--0 .stars__star:nth-child(-n + 1)' should be used to select the first star in '.stars--0'.